handlers/overview.go

```diff
@@ -368,8 +369,10 @@ func filterFlexOverview(w http.ResponseWriter, req *http.Request, f *FilterFlex,
 	}
 
 	basePage := f.Render.NewBasePageModel()
+	custom := helpers.IsBoolPtr(filterJob.Custom)
+
 	m := mapper.NewMapper(req, basePage, eb, lang, serviceMsg, filterID)
-	overview := m.CreateFilterFlexOverview(*filterJob, fDims, dimDescriptions, pops, *sdc, isMultivariate)
+	overview := m.CreateFilterFlexOverview(*filterJob, fDims, dimDescriptions, pops, *sdc, isMultivariate, custom)
```

Comment:
The `filterJob` struct is already a parameter
```suggestion
	overview := m.CreateFilterFlexOverview(*filterJob, fDims, dimDescriptions, pops, *sdc, isMultivariate)
```